README.md

```diff
@@ -28,27 +32,33 @@ foo = bar
 [font-desc]: http://www.pygtk.org/docs/pygtk/class-pangofontdescription.html
 
 ### [general]
+
 - `close_last_tab`: One of `new`, `close`, `blank`
 
 ### [keymap]
+
 - `prev_tab`: Previous tab, defaults to `Control + Page_Up`
 - `next_tab`: Next tab, defaults to `Control + Page_Down`
 - `move_tab_prev`: Move tab left, defaults to `Control + Shift + Page_Up`
 - `move_tab_next`: Move tab right, defaults to `Control + Shift + Page_Down`
 - `new_tab`: New tab, defaults to `Control + Shift + T`
 - `close_tab`: Close tab, defaults to `Control + F4`
 - `edit_tab`: Edit tab label, defaults to `Control + Shift + E`
+- `switch_to_tab_1`, `switch_to_tab_2` ... `switch_to_tab_10`: Switch to tab 1, 2 ... 10, defaults to `Alt + 1`, `Alt + 2` ... , `Alt + 0`
```

Comment:
Might as well make the commas consistent:

> `switch_to_tab_1`, `switch_to_tab_2`, ..., `switch_to_tab_10`: Switch to tab 1, 2, ..., 10, defaults to `Alt + 1`, `Alt + 2`, ..., `Alt + 0`

Reply:
Done